src/components/Reports/ReportForm.tsx

```diff
@@ -1,37 +1,96 @@
 import { Textfield } from '@terraware/web-components';
 import { Report } from 'src/types/Report';
+import strings from 'src/strings';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import useDebounce from 'src/utils/useDebounce';
+
+const DEBOUNCE_TIME_MS = 500;
 
 export type ReportFormProps = {
   editable?: boolean;
   report: Report;
   onChange?: (report: Report) => void;
+  onUpdateReport?: (field: string, value: any) => void;
 };
 
 export default function ReportForm(props: ReportFormProps): JSX.Element {
-  const { editable, report } = props;
+  const { editable, report, onUpdateReport } = props;
   const theme = useTheme();
+  const { isMobile } = useDeviceInfo();
+
+  const [summaryOfProgress, setSummaryOfProgress] = useState(report.summaryOfProgress ?? '');
+  useDebounce(summaryOfProgress, DEBOUNCE_TIME_MS, (value) => {
+    if (onUpdateReport) {
+      onUpdateReport('summaryOfProgress', value);
+    }
+  });
+
+  const [projectNotes, setProjectNotes] = useState(report.notes ?? '');
+  useDebounce(projectNotes, DEBOUNCE_TIME_MS, (value) => {
+    if (onUpdateReport) {
+      onUpdateReport('notes', value);
+    }
+  });
+
   return (
-    <Box
-      display='flex'
-      flexDirection='column'
+    <Grid
+      container
+      spacing={theme.spacing(3)}
       borderRadius={theme.spacing(3)}
-      padding={theme.spacing(3)}
+      padding={theme.spacing(0, 3, 3, 0)}
+      marginLeft={0}
       sx={{
         backgroundColor: theme.palette.TwClrBg,
       }}
     >
-      <Typography key='thing1'>Thing 1 for report {`${report.year}-Q${report.quarter}`}</Typography>
-      <Typography key='thing2'>Thing 2 for report {`${report.year}-Q${report.quarter}`}</Typography>
-      <Textfield
-        key='testInput'
-        placeholder={'Test Input...'}
-        label='Details'
-        id='testInput'
-        type='text'
-        disabled={!editable}
-      />
-    </Box>
+      <Grid item xs={12}>
+        <Typography fontSize='20px' fontWeight={600}>
+          {strings.PROJECT_INFORMATION}
+        </Typography>
+      </Grid>
+      <Grid item xs={12} marginLeft={theme.spacing(-3)} marginTop={theme.spacing(-3)}>
+        <OverviewItemCard isEditable={false} title={strings.ORGANIZATION} contents={report.organizationName ?? ''} />
+      </Grid>
+      <Grid item xs={isMobile ? 12 : 4} marginLeft={theme.spacing(-3)} marginTop={theme.spacing(-3)}>
+        <OverviewItemCard isEditable={false} title={strings.SEED_BANKS} contents={`${report.totalSeedBanks}` ?? '0'} />
+      </Grid>
+      <Grid item xs={isMobile ? 12 : 4} marginLeft={theme.spacing(-3)} marginTop={theme.spacing(-3)}>
+        <OverviewItemCard isEditable={false} title={strings.NURSERIES} contents={`${report.totalNurseries}` ?? '0'} />
+      </Grid>
+      <Grid item xs={isMobile ? 12 : 4} marginLeft={theme.spacing(-3)} marginTop={theme.spacing(-3)}>
```

Comment:
if you want to avoid the `isMobile ? 12 : 4` repetition.. extract out int a func?